packages/apps/patron-deletion-tracker/src/app.ts

```diff
@@ -1,5 +1,60 @@
+import { SierraClient } from '@weco/sierra-client';
+import { Auth0Client } from '@weco/auth0-client';
+import { ResponseStatus } from '@weco/identity-common';
 import { ScheduledHandler } from 'aws-lambda';
+import { startOfYesterday, endOfYesterday } from 'date-fns';
+import { rateLimit } from './rate-limits';
 
-export const lambdaHandler: ScheduledHandler = async (event) => {
-  console.log('I am a Patron deletion tracker');
+const auth0RateLimited = rateLimit({
+  // See https://auth0.com/docs/troubleshoot/customer-support/operational-policies/rate-limit-policy/management-api-endpoint-rate-limits#self-service-subscription-limits
+  rateLimits: {
+    minute: 200,
+    second: 20,
+  },
+  maxConcurrency: 5,
+});
+
+// The time before the Lambda will exit at which we should log an error that we
+// won't be able to complete the deletions
```

Comment:
Note: a Lambda timeout becomes an error that goes to the Lambda's DLQ, and our alerting will catch it.

Reply:
Will expand this comment to include that the intention is around logging the actual failed records